app/src/main/java/com/github/se/cyrcle/model/parking/online/ParkingRepositoryFirestore.kt

```diff
@@ -68,12 +71,19 @@ class ParkingRepositoryFirestore @Inject constructor(private val db: FirebaseFir
         .addOnFailureListener { onFailure(it) }
   }
 
-  override fun getParkingsBetween(
-      start: Point,
-      end: Point,
+  override fun getParkingsForTile(
+      tile: Tile,
       onSuccess: (List<Parking>) -> Unit,
       onFailure: (Exception) -> Unit
   ) {
+    // Get the bottom left and top right points of the tile
+    // This is a temporary solution for the time that `Parking.tile` gets added on Firestore
```

Comment:
Please mark this with a `TODO` or another easy searchable string so that we can find them later ^^